Notification Service/NotificationService.swift

```diff
@@ -18,13 +18,15 @@ class NotificationService: UNNotificationServiceExtension {
     /// The content handler we received for the notification request we're processing.
     private var contentHandler: ((UNNotificationContent) -> Void)?
 
-    private var recipients: [INPerson] = []
+    private var author: INPerson?
     private var conversation: ChatChannel?
     private var message: ChatMessage?
-    private var author: INPerson?
-    private var conversationId: String?
-
-    private var chatHandler: ChatRemoteNotificationHandler?
```

Comment:
No need for a member reference to the chat handler any more. Adding a convenience var for getting the message context.